tools/ncc/skip.yml

```diff
@@ -30,3 +30,4 @@ i_activity: "Allow original naming from beta"
 i_actor: "Allow original naming from beta"
 score: "Allow original naming from beta"
 c_LOCATIONS_NUM: "Allow original naming from beta"
+SDLCALL: "SDL function call convention"
```

Comment:
This should go to the --definition parameter of the ncc invocation instead like the others such as WINAPI